scripts/lib/install-node

```diff
@@ -20,20 +20,23 @@ if node_wrapper_path="$(type -p node)"; then
     current_node_version="$(node --version)"
 fi
 
-if [ "$($YARN_BIN --version 2>/dev/null)" = "$yarn_version" ] && [ "$current_node_version" = "v$node_version" ] && [ -L "$node_wrapper_path" ]; then
+if [[ "$($YARN_BIN --version 2>/dev/null)" == "$yarn_version" \
+          && "$current_node_version" == "v$node_version" \
+          && -L "$node_wrapper_path" ]]; then
     echo "Node version $node_version and yarn version $yarn_version are already installed."
     exit 0
 fi
 
-if [ "$current_node_version" != "v$node_version" ] || ! [ -L "$node_wrapper_path" ]; then
+if [[ "$current_node_version" != "v$node_version" || ! -L "$node_wrapper_path" ]]; then
     export NVM_DIR=/usr/local/nvm
-    if ! [ -e "$NVM_DIR/nvm.sh" ]; then
-        wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+    if  [[ ! -e "$NVM_DIR/nvm.sh" ]]; then
+        wget -O- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
```

Comment:
Why the removal of `-q` here?

Reply:
The commit message explained “don't mask wget stderr output with -q”. Not wrong. Another possibility is `-nv` since the defaults are pretty noisy.

Reply:
With a self-signed certificate this is the first place the installation crashed, with very little help. Because of no `pipefail` this continued to run and crashed a couple lines later with no reasonable error message. `-nv` sounds fine too :)